dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -50,6 +50,11 @@
  *         <td>The hostname to bind to.</td>
  *     </tr>
  *     <tr>
+ *         <td>{@code inheritChannel}</td>
+ *         <td>false</td>
+ *         <td>If your application launch by the <a href="https://github.com/kazuho/p5-Server-Starter">Server::Starter</a>, please set 'true'</td>
```

Comment:
Also, please don't forget to [rebase and squash](http://rebaseandsqua.sh) your commits.